src/cli/schema/applications/appconfig_app.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "application": {
```

Comment:
Application configuration can be found on confluence (interface specification). The "application" is not part of it. Actually this was already reported as a bug for the creation of this app config initial file.